### ISSUE TITLE
Can't add orbit-db to yarn project

### ISSUE THREAD

**Issue by spaceywolfi** [orbitdb/orbit-db#235]
when trying to add orbit-db with yarn i get these errors:

info fsevents@1.1.2: The platform "linux" is incompatible with this module.
info "fsevents@1.1.2" is an optional dependency and failed compatibility check. Excluding it from installation.
error ipfs-log@2.0.0: The engine "node" is incompatible with this module. Expected version "^6.x.x".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.

**Comment by haadcode** [comment]
Which version of Node.js are you on?

**Comment by spaceywolfi** [comment]
i'm using v8.5.0

**Comment by spaceywolfi** [comment]
Is anyone able to replicate the error?

```bash
mkdir test
npm init -y
yarn add orbit-db
```

**Comment by haadcode** [comment]
Got it. I don't use Yarn myself and have never come across this problem, but I believe [this is the line causing it](https://github.com/haadcode/ipfs-log/blob/v2.0.0/package.json#L13).

There's a new version of `ipfs-log` coming out soon (v3.0.0) which should fix this.

**Comment by millette** [comment]
Meanwhile, this should work:

```
yarn --ignore-engines add orbit-db
```

**Comment by haadcode** [comment]
@millette thanks for sharing the workaround!

Indeed 3.0 was released, meant 4.0 :) Fixed it in my comment.

**Comment by haadcode** [comment]
This should be fixed now in [v0.18.0](https://www.npmjs.com/package/orbit-db). I'm able to run `yarn add orbit-db` without errors.

Closing as fixed, but if anyone still has the problem, please re-open the issue and let us know!

Thank you everyone for helping out on this one! ❤️
